qkeras/quantizers.py

```diff
@@ -1706,6 +1706,80 @@ def get_config(self):
     }
     return config
 
+class quantized_sigmoid(BaseQuantizer):  # pylint: disable=invalid-name
+  """Computes a quantized sigmoid to a number of bits.
+
+  Modified from:
+
+  [https://github.com/BertMoons/QuantizedNeuralNetworks-Keras-Tensorflow]
+
+  Attributes:
+    bits: number of bits to perform quantization.
+    integer: number of bits to the left of the decimal point.
+    symmetric: if true, we will have the same number of values for positive
+               and negative numbers.
+    use_stochastic_rounding: if true, we perform stochastic rounding.
+
+  Returns:
+    Function that performs sigmoid + quantization to bits in the range 0.0 to 1.0.
+  """
+
+  def __init__(self, bits=8, integer=0, symmetric=0,
+               use_stochastic_rounding=False):
+    super(quantized_sigmoid, self).__init__()
+    self.bits = bits
+    self.integer = integer
+    self.symmetric = symmetric
+    self.use_stochastic_rounding = use_stochastic_rounding
+
+  def __str__(self):
+    flags = [str(self.bits), str(self.integer)]
+    if self.symmetric or self.use_stochastic_rounding:
+      flags.append(str(int(self.symmetric)))
+    if self.use_stochastic_rounding:
+      flags.append(str(int(self.use_stochastic_rounding)))
+    return "quantized_sigmoid(" + ",".join(flags) + ")"
+
+  def __call__(self, x):
+    non_sign_bits = self.bits - 1
+    m = pow(2, non_sign_bits)
+    m_i = pow(2, self.integer)
+    p = _sigmoid(x / m_i) * m
+    xq = m_i * tf.keras.backend.clip(
```

Comment:
Please use `K` instead of `tf.keras.backend`

Reply:
Changed to `K`. `tf.keras.backend` is still used in most other parts in the file instead of `K`.